cli/src/main.rs

```diff
@@ -74,6 +79,33 @@ async fn main() -> Result<()> {
         .await?;
 
     match opts.command {
+        Command::Cleanup => {
+            let state = get_world_state(nats.clone()).await?;
+            let stream = nats.jetstream.get_stream("plane_state").await.unwrap();
+
+            for (cluster_name, cluster) in &state.clusters {
+                for (backend_id, backend) in &cluster.backends {
+                    let Some((timestamp, state)) = backend.state_timestamp() else { continue };
+
+                    if state == BackendState::Swept {
+                        println!(
+                            "Removing backend {} from cluster {}, swept at {}",
+                            backend_id, cluster_name, timestamp
+                        );
+
+                        stream
+                            .purge()
+                            .filter(format!(
+                                "state.cluster.{}.backend.{}.>",
```

Comment:
hmmmm so this goes through the entire stream for each swept backend? wouldn't it be faster to construct a filter for all the relevant backends then purge().fliter() once? Not sure if that's possible.

Reply:
Unfortunately NATS doesn't allow this. I'm not too worried about the performance of this particular task, though; it's something that will eventually be run in a loop.